index.html

```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
+                <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                    <form action="#" method="post">
+                        <fieldset class="mobile-contact">
+                            <label for="username">
+                                <input type="text" class="input" required name="username" maxlength="30" placeholder="Username">
+                            </label>
+                            <label for="email">
+                                <input type="email" class="input" required name="email" placeholder="name@domain.com">
+                            </label>
+                            <label for="message">
+                                <textarea name="message" class="message-mobile" required maxlength="500" placeholder="Write your message here"></textarea>
+                            </label>
```

Comment:
Please fix the font issues here. Kindly try to follow the Figma template. 👍
<img width="413" alt="Screenshot 2023-01-18 at 12 42 13 AM" src="https://user-images.githubusercontent.com/25887253/213020583-7a1a10a7-a959-4369-82fd-97fa0def230c.png">
<img width="413" alt="Screenshot 2023-01-18 at 12 42 26 AM" src="https://user-images.githubusercontent.com/25887253/213020585-d847e49c-5194-4151-9e16-42820ff6a012.png">

Reply:
Please 🙏  can you explain more? What are the font issues?

Reply:
Thanks for the review @huseyinbiyik